Test.swift

```diff
@@ -1,12 +1,11 @@
 struct TestView: View {
   var body: some View {
-    Text("green")
-    Text("green")
-    Text("green")
-    Text("green")
-    Text("green")
-    Text("hyerin")
-    Text("hyerin")
-    Text("hyerin")
+    VStack {
+      Text("green")
+      
+      Text("test PR")
+      
+      Text("test PR")
```

Comment:
```suggestion
      Text("green PR")
```
test PR 텍스트는 중복인데 green PR로 텍스트를 변경하면 어떨까요?